docs/victory-theme/ecology.md

```diff
@@ -0,0 +1,302 @@
+
+VictoryTheme works across all component types - it can be applied to chart components themselves.
+
+``` playground
+<svg style={{width: "100%", height: "100%"}} viewBox="0 25 350 300">
+  <VictoryAxis
+    theme={VictoryTheme.material}
+    standalone={false}
+  />
+  <VictoryCandlestick
+    theme={VictoryTheme.material}
+    standalone={false}
+    domainPadding={5}
+  />
+</svg>
+```
+
+It can be applied to VictoryStack or VictoryGroup, and then passed down to wrapped chart components.
+
+```playground
+<VictoryStack theme={VictoryTheme.material}>
+  {range(6).map((index) => {return <VictoryArea key={index}/>})}
+</VictoryStack>
+```
+
+It can even be applied to [VictoryChart][] and passed down to all subsequent child components.
+
+```playground
+<VictoryChart theme={VictoryTheme.material}>
+  <VictoryGroup
+    height={500}
+    offset={20}
+  >
+    <VictoryBar
+      data={[
+        {x: 1, y: 1},
+        {x: 2, y: 2},
+        {x: 3, y: 3}
+      ]}
+    />
+    <VictoryBar
+      data={[
+        {x: 1, y: 2},
+        {x: 2, y: 1},
+        {x: 3, y: 1}
+      ]}
+    />
+    <VictoryBar
+      data={[
+        {x: 1, y: 3},
+        {x: 2, y: 4},
+        {x: 3, y: 2}
+      ]}
+    />
+  </VictoryGroup>
+</VictoryChart>
+```
+
+### Create Your Own Custom Theme Object
+
+Most chart types only need styles from a theme object, like [VictoryScatter][] and [VictoryLine][], below.
+
+```playground_norender
+class App extends React.Component {
+  render() {
+    const myTheme = {
+      scatter: {
+        data: {
+          fill: "red"
+        },
+        labels: {
+          padding: -25,
+          fontFamily: "Garamond"
+        },
+        // even if one of the subobjects isn't being utilized,
+        // it needs to exist for the theme to be supported
+        parent: {}
+      },
+      line: {
+        data: {
+          stroke: "red",
+          fill: "none"
+        },
+        labels: {
+          padding: 15,
+          fontFamily: "Garamond"
+        },
+        parent: {}
+      }
+    };
+
+    const data = [
+      {x: 1, y: 1},
+      {x: 2, y: 4},
+      {x: 3, y: 2},
+      {x: 4, y: 6},
+      {x: 5, y: 5},
+      {x: 6, y: 9}
+    ];
+
+    return (
+      <svg style={{width: "100%", height: "100%"}} viewBox="0 25 350 300">
+        <VictoryScatter
+          data={data}
+          size={7}
+          theme={myTheme}
+          labels={["1", "2", "3", "4", "5", "6"]}
+        />
+        <VictoryLine
+          data={data}
+          theme={myTheme}
+          label="Hello"
+        />
+      </svg>
+    )
+  }
+}
+
+ReactDOM.render(<App/>, mountNode);
+```
+
+[VictoryCandlestick][] has a special theme object with both style and props sub-objects. This is how you alter VictoryCandlestick's ```candleColors``` prop within a theme.
+
+```playground_norender
+class App extends React.Component {
+  render() {
+    const myTheme = {
+      candlestick: {
+        style: {
```

Comment:
yes - right now, only candlestick does because it's the only one that needs special props compared to the other types (since candlecolors are props). I could make this consistent by nesting all of the other chart types within a style object and then adding an empty props object? or i could get rid of overall theme props and add them individually to each chart type? that seems repetitive though